### DISCUSSION TITLE
Actions column location in table with actions

### DISCUSSION THREAD

**Discussion by grant-sea** [helloflask/bootstrap-flask#286] [Q&A]
For the table with actions, can I have the actions column in the left (1st colcumn) of the table instead of right? As you know, some time the table is too wide and is respensive to the page. Then the actions cannot be seen before moving the scroll bar. Thanks!

**Comment by PanderMusubi** [comment]
Would you like to make a PR for this via an optional parameter?

**Reply by grant-sea** [reply]
Thanks for the invitation. I'm a new hand of GitHub. Just signed up today! Programming is my hobby and I know a little about HTML. But I tried in my side, it can work. I belive my code is just functional but not perfect. Will try my first PR!

**Comment by PanderMusubi** [comment]
Yes, try it, we can help you if you get stuck.
